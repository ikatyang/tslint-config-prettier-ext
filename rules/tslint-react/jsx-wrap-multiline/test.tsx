const badMultiline = <div
    className="my-class"
    tabIndex={-1}
>
    {children}
</div>;
