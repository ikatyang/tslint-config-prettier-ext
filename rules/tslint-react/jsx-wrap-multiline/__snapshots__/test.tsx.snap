// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const badMultiline = <div
    className=\\"my-class\\"
    tabIndex={-1}
>
    {children}
</div>;

======
const badMultiline = (
  <div className=\\"my-class\\" tabIndex={-1}>
    {children}
  </div>
);

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (jsx-wrap-multiline) /rules/tslint-react/jsx-wrap-multiline/test.tsx[1, 22]: Multiline JSX elements must be wrapped in parentheses
"
`;
